tailwind.config.ts

```diff
@@ -13,6 +13,30 @@ const config: Config = {
         "gradient-conic":
           "conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
       },
+      colors: {
+        primary: {
+          500: '#42BBFF',
+          800: '#0C3247',
+        },
```

Comment:
```suggestion
        primary: {
          500: '#42BBFF',
          800: '#0C3247',
          900: '#42BBFF',
        },
```

For the colors with opacity- you can approach this in a lot of different ways.

Because primary-500 and primary-900 are the same color: You could technically just write "primary-500/10" in your **className** to indicate you want a specific level of opacity for a color.

To explicitly add it in your config with the opacity included (recommended), you can either:

1) use the last 2 values of a hexadecimal color to indicate the opacity:
```
900: "#42BBFF1A",
```

2) use any of the other ways to represent color in CSS that might be more explicit:
```
900: "rgba(66, 187, 255, 0.1)",
```

Reply:
![image](https://github.com/DDVVPP/GitNote/assets/4108484/88ab79c5-c2e1-43a3-b4e6-7ab87f74f361)

Pressing the drop-down labeled "Hex" here in the color menu will give you different options if you would prefer.

Reply:
Thank you!